### ISSUE TITLE
WAVE errors datatable

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/highcharts-editor#96]
## Description

Might be a regression bug but highest priority to fix all the WAVE errors associated with the data table. There are missing form labels tied to every datatable cell and empty table headers on each labelled datatable header cell. 

Also, ensure that the datatable header text is bold as currently is inherits the styling which will transform when used in a different host application. 

## Steps to reproduce

1. Upload a data file and navigate to datatable view
2. Run WAVE tool and find a collection of errors for each non-empty datatable cell
